src/ui/components.tsx

```diff
@@ -176,10 +176,10 @@ export const CheckMkSelectNegatable = <T extends RequestSpecNegatableOptionKeys>
           label={label}
           autocompleter={autocompleter}
           onChange={onValueChange}
-          value={value.value}
+          value={value.value === '' ? undefined : value.value}
```

Comment:
ah! sorry, to mention this so late, but could there be another/better place to fix this?
why is the value an empty string and not undefined in the first place? could this be fixable with a better default value andtyping?

Reply:
can you give it a try? if it's too much work, or looks too ugly, we can still go back to this option.

Reply:
Thank you for pushing me, it changed exaclty nothing.